book/src/livenessprobe.md

```diff
@@ -7,7 +7,7 @@
 **Status:** GA/Stable
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) |
---|--|--|--|--|--
+--|--|--|--|--|--|--
 [livenessprobe v2.0.0](https://github.com/kubernetes-csi/livenessprobe/releases/tag/v2.0.0) | [release-2.0](https://github.com/kubernetes-csi/livenessprobe/tree/release-2.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/livenessprobe:v2.0.0 | v1.13 | -
```

Comment:
All of these rows are missing an entry for "Max CSI version"

Reply:
@msau42 Thanks for the review Added Max CSI version